src/irc_aliases.lua

```diff
@@ -280,7 +280,10 @@ aliases['rot13%s.*'] = function (ms, c, t, msg)
 end
 
 aliases['restart'] = function (_, _, _, _, authed)
-    if authed then return true end
+    if authed then
+        print('restarting')
```

Comment:
if this is going to be `print`, it should be `debug`-gated; otherwise, it should probably be a `privmsg` resonse.